lib/DatabaseLayer.py

```diff
@@ -78,6 +78,10 @@ def updateCVE(cve):
         {"id": cve["id"]},
         {
             "$set": {
+                "cvss3": cve['cvss3'],
+                "impact3": cve['impact3'],
+                "exploitability3": cve['exploitability3'],
+                "cvss3-vector": cve['cvss3-vector'],
```

Comment:
Another check for the presence of cvss3 should be made before this update statement is constructed